pkg/sbom/implementation.go

```diff
@@ -90,6 +93,7 @@ func GenerateAPKPackage(spec *Spec) (pkg, error) {
 		Copyright:        spec.Copyright,
 		Namespace:        spec.Namespace,
 		Arch:             spec.Arch,
+		Originator:       "Organization: " + cases.Upper(language.English).String(spec.Namespace),
```

Comment:
I think this is wrong, this should be the supplier:

```suggestion
		Supplier:       "Organization: " + cases.Upper(language.English).String(spec.Namespace),
```

The originator should be the maintainer of the package listed in the APK ([see the field here, for example](https://pkgs.alpinelinux.org/package/edge/main/x86/cjson)). Unfortunately, I think we are not populating those in wolfi (which is something we should fix).

Reply:
https://pkg.go.dev/chainguard.dev/apko/pkg/sbom/generator/spdx#Package doesn't have a `Supplier`, should I add one?

Reply:
Better yet, should I switch apko+melange to use something like https://pkg.go.dev/github.com/spdx/tools-golang@v0.5.3/spdx/v2/v2_3#Package ?

Reply:
If we are changing the custom types, I would vote to switch the innards of all of our SBOM tools to protobom, that's why we created it (and it uses the spdx go tools to render the doc).

Reply:
Sounds good. I'll leave that for now, but we I like that plan.